### PR TITLE
Fix command options in CLI and web help

### LINKED ISSUE #927: Web help is missing options for root "zowe" command
Would be nice to have options on the root command like `--version` and `--available-commands` included.

Thanks @anaxceron for reporting this 🙂

### PR DESCRIPTION
**Pull request by t1m0thyj**
⚠️ This PR makes changes to CLI help which may break a lot of snapshots across integration tests for Zowe CLI and plug-ins.
* [ ] Update snapshots in Zowe plugins
* [ ] Inform internal teams about snapshot changes
* [ ] Create `wont-fix` issue for other plugins to reference

Resolves #927 and #928

Also removes the timestamp from web help. It did not provide much value and prevented us from having reproducible builds.

### CODE REVIEW

**Comment by t1m0thyj** [comment]
> ⚠️ This PR makes changes to CLI help which may break a lot of snapshots across integration tests for Zowe CLI and plug-ins.
> 
> Resolves #927 and #928
> 
> Also removes the timestamp from web help. It did not provide much value and prevented us from having reproducible builds.

**Review by gejohnston — APPROVED** [review]

**Review by ATorrise — APPROVED** [review]

**Review by awharn — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
LGTM 😋 
Do we plan to address the checklist in this PR description after merging, or do we plan to have draft PRs opened with the updates already implemented (as well as open the issue) **before** merging this?

File: packages/cmd/src/help/WebHelpGenerator.ts [hunk 1]
```diff
@@ -392,14 +392,14 @@ export class WebHelpGenerator {
      */
     private writeTreeData() {
         const treeDataPath = path.join(this.mDocsDir, "..", "tree-data.js");
-        // eslint-disable-next-line @typescript-eslint/no-magic-numbers
-        const tzOffset = new Date().getTimezoneOffset() * 60000;
-        // eslint-disable-next-line @typescript-eslint/no-magic-numbers
-        const dateString = new Date(Date.now() - tzOffset).toISOString().replace("T", " ").slice(0, -5);
+        let footerStr = `${this.mConfig.callerPackageJson.name} ${this.mConfig.callerPackageJson.version}`;
+        if (process.env.NODE_ENV === "development") {
+            footerStr += " (dev)";
```
[the remark this answers is not in the document]
Comment (t1m0thyj) [reply packages/cmd/src/help/WebHelpGenerator.ts:397]: Good catch, I think checking the value of `process.env.NODE_ENV` was a bad idea here. Updated this to look for the presence of a `.git` folder instead, which also fixes missing coverage 🙂 

This should make it easy to distinguish between:
* `@zowe/cli 7.10.1` - web help built from CLI 7.10.1 installed from NPM
* `@zowe/cli 7.10.1 (dev)` - web help built from source (may include local changes)

**Comment by t1m0thyj** [comment]
> LGTM 😋 Do we plan to address the checklist in this PR description after merging, or do we plan to have draft PRs opened with the updates already implemented (as well as open the issue) **before** merging this?

I think we should address these items after merging the PR, since its not possible to update the snapshots in CLI plugins until these changes have been merged into Imperative and CLI. Added tasks to our internal story for tracking the remaining items.

**Pull request merged**